app/gateway/2.7.x/vitals/vitals-estimates.md

```diff
@@ -0,0 +1,175 @@
+vitals_codes_by_cluster
+[second(t), 1, 200, 2]
+[minute(t), 60, 200, 2]
+[day(t), 84600, 200, 2]
+
+vitals_codes_by_workspace
+[workspace_id, second(t), 1, 200, 2]
+[workspace_id, minute(t), 60, 200, 2]
+[workspace_id, day(t), 84600, 200, 2]
+
+vitals_codes_by_route
+[service_id, route_id, second(t), 1, 200, 2]
+[service_id, route_id, minute(t), 60, 200, 2]
+[service_id, route_id, day(t), 84600, 200, 2]
+```
+
+### Scenario where new rows are inserted
+
+If the last request received a 500 status code, new rows are inserted:
+
+```bash
+vitals_codes_by_cluster
+[second(t), 1, 200, 1]
+[minute(t), 60, 200, 1]
+[day(t), 84600, 200, 1]
+
+[second(t), 1, 500, 1]
+[minute(t), 60, 500, 1]
+[day(t), 84600, 500, 1]
+
+vitals_codes_by_workspace
+[workspace_id, second(t), 1, 200, 1]
+[workspace_id, minute(t), 60, 200, 1]
+[workspace_id, day(t), 84600, 200, 1]
+
+[workspace_id, second(t), 1, 500, 1]
+[workspace_id, minute(t), 60, 500, 1]
+[workspace_id, day(t), 84600, 500, 1]
+
+vitals_codes_by_route
+[service_id, route_id, second(t), 1, 200, 1]
+[service_id, route_id, minute(t), 60, 200, 1]
+[service_id, route_id, day(t), 84600, 200, 1]
+
+[service_id, route_id, second(t), 1, 500, 1]
+[service_id, route_id, minute(t), 60, 500, 1]
+[service_id, route_id, day(t), 84600, 500, 1]
+```
+
+### Scenario where a second row is inserted
+
+Assume that at `t + 5s`, where `minute(t)==minute(t + 5s)`, Kong Gateway proxies the same request returning 200. Since `minute()` and `day()` for both `t` and `t + 5s` are the same, minute and day rows should just be updated. Since `second()` is different for the two instants, a new second row should be inserted in each table.
+
+```bash
+vitals_codes_by_cluster
+[second(t), 1, 200, 1]
+[second(t + 5s), 1, 200, 1]
+[minute(t), 60, 200, 2]
+[day(t), 84600, 200, 2]
+
+vitals_codes_by_workspace
+[workspace_id, second(t), 1, 200, 1]
+[workspace_id, second(t + 5s), 1, 200, 1]
+[workspace_id, minute(t), 60, 200, 2]
+[workspace_id, day(t), 84600, 200, 2]
+
+vitals_codes_by_route
+[service_id, route_id, second(t), 1, 200, 1]
+[service_id, route_id, second(t + 5s), 1, 200, 1]
+[service_id, route_id, minute(t), 60, 200, 2]
+[service_id, route_id, day(t), 84600, 200, 2]
+```
+
+In summary, the number of rows in those status codes tables is directly related to:
+
+* The number of status codes observed in Kong Gateway proxied requests
+* The number of Kong Gateway entities involved in those requests
+* The constant flow of proxied requests
+
+In an estimate of row numbers in scenario, consider a Kong Gateway cluster with the following characteristics:  
+
+* A constant flow of requests returning all 5 possible groups of status codes (1xx, 2xx, 3xx, 4xx and 5xx).
+* Just 1 workspace, 1 service, and 1 route
+
+After 24 hours of traffic, the status codes tables will have this number of rows:
+
+* **vitals_codes_by_cluster**: 5 row for day, (5 * 24 * 60 = 7200) rows for minutes and (5 * 60 * 60 = 18000) rows for seconds. Total = 25200 rows.
```

Comment:
This would be better presented in a table, with the headings: status code table | day | minute | seconds | total